python/src/endpoints/db_delete.py

```
@@ -1,10 +1,63 @@
+def delete_old_completed_tasks() -> None:
+    """Delete old completed tasks with multiple conditions."""
+    # Initialize the client with your API key
+    client = Worqhat(
+        api_key=os.environ.get("WORQHAT_API_KEY"),
+        environment=os.environ.get("WORQHAT_ENVIRONMENT", "production"),  # Defaults to production
+    )
+
+    try:
+        # Get date from 30 days ago
+        from datetime import datetime, timedelta
+        thirty_days_ago = (datetime.now() - timedelta(days=30)).isoformat()
```

Ruff (0.14.1)

Local variable `thirty_days_ago` is assigned to but never used

Remove assignment to unused variable `thirty_days_ago`

(F841)